### PR TITLE
BXMSDOC-8415-main: Add proxy optional step

### PR DESCRIPTION
**Pull request by csherrar**

### CODE REVIEW

**Review by spolti — APPROVED** [review]

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc [hunk 1]
```diff
@@ -70,6 +70,14 @@ ifdef::PAM[]
 In the simplified mode, {CENTRAL} Monitoring does not require a persistent volume claim. You can use this mode in environments that do not support `ReadWriteMany` access to persistent storage. You can not use {CENTRAL} Monitoring in the simplified mode to design custom dashboards.
 +
 endif::PAM[]
+** Optional: If you want to configure the proxy settings, use the following environmental variables:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:73]:
```suggestion
** Optional: If you want to configure the proxy settings, use the following environment variables:
```

**Review by msmagnanijr — APPROVED** [review]
lgtm

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc [hunk 2]
```diff
@@ -70,6 +70,14 @@ ifdef::PAM[]
...
+*** `https_proxy`: The location of the https proxy. This takes precedence over `HTTPS_PROXY`, `http_proxy`, and `HTTP_PROXY`, and will be used for both Maven builds and Java runtime. For example: `myuser:mypass@127.0.0.1:8080`.
+*** `HTTPS_PROXY`: The location of the https proxy. This takes precedence over `http_proxy` and `HTTP_PROXY`, and will be used for both Maven builds and Java runtime. For example: `myuser@127.0.0.1:8080`.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:75]:
```suggestion
*** `HTTPS_PROXY`: The location of the https proxy. This takes precedence over `http_proxy` and `HTTP_PROXY`, and is used for both Maven builds and Java runtime. For example: `myuser@127.0.0.1:8080`.
```

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc [hunk 3]
```diff
@@ -70,6 +70,14 @@ ifdef::PAM[]
...
+*** `http_proxy`: The location of the http proxy. This takes precedence over `HTTP_PROXY` and will be used for both Maven builds and Java runtime. For example: `http://127.0.0.1:8080`.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:76]:
```suggestion
*** `http_proxy`: The location of the http proxy. This takes precedence over `HTTP_PROXY` and is used for both Maven builds and Java runtime. For example: `http://127.0.0.1:8080`.
```

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc [hunk 4]
```diff
@@ -70,6 +70,14 @@ ifdef::PAM[]
...
+*** `HTTP_PROXY`: The location of the http proxy. This will be used for both Maven builds and Java runtime. For example: `127.0.0.1:8080`.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:77]:
```suggestion
*** `HTTP_PROXY`: The location of the http proxy. This is used for both Maven builds and Java runtime. For example: `127.0.0.1:8080`.
```

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc [hunk 5]
```diff
@@ -70,6 +70,14 @@ ifdef::PAM[]
...
+*** `no_proxy`: A comma separated lists of hosts, IP addresses or domains that can be accessed directly. This takes precedence over `NO_PROXY` and will be used for both Maven builds and Java runtime. For example: `*.example.com`.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:78]:
```suggestion
*** `no_proxy`: A comma separated lists of hosts, IP addresses, or domains that can be accessed directly. This takes precedence over `NO_PROXY` and is used for both Maven builds and Java runtime. For example: `*.example.com`.
```

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc [hunk 6]
```diff
@@ -70,6 +70,14 @@ ifdef::PAM[]
...
+*** `NO_PROXY`: A comma separated lists of hosts, IP addresses or domains that can be accessed directly. This will be used for both Maven builds and Java runtime. For example: `foo.example.com,bar.example.com`.
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:79]:
```suggestion
*** `NO_PROXY`: A comma separated lists of hosts, IP addresses, or domains that can be accessed directly. This is used for both Maven builds and Java runtime. For example: `foo.example.com,bar.example.com`.
```

File: doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc [hunk 7]
```diff
@@ -150,7 +150,16 @@ If you are using the EntrepriseDB Postgres database server, use an URL starting
 If you use a MySQL version 8 image provided by Red Hat on {OPENSHIFT}, to enable the plugin, set the `MYSQL_DEFAULT_AUTHENTICATION_PLUGIN` environment variable to `mysql_native_password`.
 +
 If you create users on the MySQL version 8 server before enabling the `mysql_native_password` plugin, you must update the `mysql-user` table after you enable the plugin.
++
 endif::PAM[]
+. Optional: If you want to configure the proxy settings, use the following environmental variables:
```
Comment (emmurphy1) [inline doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc:155]: Same comments as the previous section.

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc (hunk 1 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:73]: Done

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc (hunk 2 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:75]: Done

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc (hunk 4 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:77]: Done

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc (hunk 6 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:79]: Done

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc (hunk 5 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:78]: Done

File: doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc (hunk 3 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc:76]: Done

File: doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc (hunk 7 above)
Comment (csherrar) [reply doc-content/enterprise-only/openshift/operator-deploy-kieserver-proc.adoc:155]: Done, Thanks @emmurphy1 !

**Pull request merged**